docs/manual/user_guide/siteonlyapi_tutorial.ipynb

```diff
@@ -0,0 +1,467 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Diasaggregate your Home/Building Mains Meter Data\n",
+    "\n",
+    "### This notebook demonstrates the use of siteonlyapi - a new NILMTK interface which is an modification of NILMTK's ExperimentAPI. It allows NILMTK users to get their home/buildings energy demands for different potential appliances.\n",
+    "\n",
+    "Lets us start with a very simple experiment to demonstrate the use of this API. This experiment shows how the user can convert their meter data into proper REDD format, and call the API  to disaggregate the energy into appliance demands based on the training set.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "from nilmtk.datastore import Key\n",
+    "from nilmtk.utils import check_directory_exists, get_datastore, get_module_directory\n",
+    "from nilm_metadata import convert_yaml_to_hdf5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Convert meter data into proper format"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def convert_data(file_path):\n",
```

Comment:
In this PR, you also added the `caxe` converter to NILMTK. Why don't you just call it? :)